src/components/OrderDetails/index.js

```diff
@@ -133,18 +173,31 @@ export const OrderDetails = (props) => {
       const order = error ? null : result
       const err = error ? result : null
       let businessData = null
+      let driversGroupData = null
       try {
         const { content } = await ordering.setAccessToken(token).businesses(order.business_id).select(propsToFetch).get({ cancelToken: source })
         businessData = content.result
         content.error && err.push(content.result[0])
       } catch (e) {
         err.push(e.message)
       }
+
+      if(user.level === 2 && order.delivery_type === 1) {
+        try {
+          const { content } =  await ordering.setAccessToken(token).driversgroups().get();
+          driversGroupData = content.result
+          content.error && err.push(content.result[0])
+        } catch(e) {
+          err.push(e.message)
+        }
+      }
+
       setOrderState({
         ...orderState,
         loading: false,
         order,
         businessData,
+        driversGroupsData: !driversGroupData ? {} : driversGroupData,
```

Comment:
you can replace the ternary with this expression 

`driversGroupsData: driversGroupData || {},`

Or,

define `let driversGroupData = {}`

and in the setOrderState only declare

`setOrderState({
        ...orderState,
        loading: false,
        order,
        businessData,
        driversGroupsData })`
        
is the same thing